index.Rmd

```diff
@@ -140,6 +140,8 @@ names(wagLandsat) <- c("band1", "band2", "band3", "band4", "band5", "band6", "ba
 plotRGB(wagLandsat, 5, 4, 3) 
 ```
 
+We have chosen to visualize the Landsat image as a false color composite, meaning that the chosen bands do not match the RGB channels. Indeed, we have plotted the near-infrared band as red, the red as green, and the green as blue.
```

Comment:
The color of the image could be a bit confusing to students without any explanation. I added one line, but I don't know if that complicates things more.

Reply:
No, it's helpful, thanks!